src/script/script.go

```diff
@@ -44,6 +58,12 @@ func PrintCommand(cmd ...string) {
 // RunCommand executes the given command-line operation.
 func RunCommand(cmd ...string) error {
 	PrintCommand(cmd...)
+	if dryrun.IsActive() {
```

Comment:
We aren't actually running any commands. We need to "fake" the checkouts in order to print out the proper branch with the commands. Thats the only thing that is happening here